docs/guides/StandarizedUnitsNaming.md

```diff
@@ -0,0 +1,115 @@
+# Standardowe nazewnictwo jednostek
+
+
+
+## Wstęp + rozpiska komunikacji radiowej
+
+W Armaforces przyjęło się stosowanie zwyczajowego, standardowego nazewnictwa jednostek. Należy tu zaznaczyć, że zwyczajowe nazewnictwo nie jest obowiązkowe (nie jest wymuszane na missionmakerach). Missionmaker może wprowadzić inne nazewnictwo w swojej misji. 
+
+Przykładowo, w przeszłosci występowały różnice w nazewnictwie, np.:
+- jednostki były nazywane zupełnie inaczej, np. w celu zachowania klimatu misji (np. Alpha -> Anton, Bravo -> Boris, itd.),
+- rzadziej używane jednostki (inne niż np. Alpha, Bravo, Kilo, November, Zulu, itd.) miały inne funkcje niż zazwyczaj (np. Mike był transportem lądowym zamiast powietrznym).
```

Comment:
I think I don't get it. Could you rewrite this somehow or explain to me what you wanted to say here?

Reply:
It seems like past mission references was not a good idea. Rephrased, I hope that now it's more readable.